tests/components/scene/stats.test.js

```diff
@@ -11,14 +9,14 @@ suite('stats', function () {
 
     el.addEventListener('loaded', function () { done(); });
 
-    utils.getUrlParameter = function () { return 'false'; };
+    window.history.replaceState(null, '', originalLocationUrl + '?stats=false');
   });
 
   teardown(function () {
     var el = this.sceneEl;
     el.parentNode.removeChild(el);
 
-    utils.getUrlParameter = originalGetUrlParameter;
+    window.history.replaceState(null, '', originalLocationUrl);
```

Comment:
Instead of trying to overwrite the `getUrlParameter` util (which is now read-only) the URL is actually rewritten using the history API.